storybookjs/storybook
# Update Docs to use new Notes form and use yarn for addon install

Issue: #3305 

## What I did
Updated documents

## How to test
View help documents and verify they look acceptable

Is this testable with Jest or Chromatic screenshots?
no?

Does this need a new example in the kitchen sink apps?
no

Does this need an update to the documentation?
N/A, this is a documentation update

If your answer is yes to any of these, please make sure to include it in your PR.
